DOCS-2950 Add static IP configuration instructions to NextCloud article (#1298)
truenas/documentation: content/SCALE/Apps/NextcloudSCALE.md

---
title: "Nextcloud"
weight: 26
---

## NextCloud Plugin

The [NextCloud](https://nextcloud.com/) plugin is a suite of client-server software for creating and using file hosting services. 

### Plugin Installation

{{< hint info >}}
**Official Applications**
 
Official containers are pre-configured to only require a name during deployment.
{{< /hint >}}

> Go to **Apps** and click the *INSTALL* button.  Enter an *Application Name* and click *Next*.

Reviewer: ```suggestion
Go to **Apps** and click the **INSTALL* button.  Enter an **Application Name** and click **Next**.
```

![SCALENextcloudInstall](/images/SCALE/SCALENextcloudInstall.png "SCALE Nextcloud Install")

> For *Nextcloud Configuration* the *Username* defaults to *admin* and *Password* defaults to *changeme*.  It is recommended that you change both of these fields to entries specific to you as the user.  Ensure you employ a password that is both strong and can be remembered.

Reviewer: ```suggestion
For **Nextcloud Configuration** the **Username** value defaults to **admin** and the **Password** value defaults to **changeme**.  We recommend that you change both of these fields to entries specific to you as the user.  Ensure you employ a password that is both strong and memberable.
```

![SCALENextcloudConfiguration](/images/SCALE/SCALENextcloudConfiguration.png "SCALE Nextcloud Configuration")

> For *Storage*, *Scaling/Upgrading Policy* and *Advanced DNS Settings* the defaults can be used for basic installation by clicking *Next* or bypassing and going directly to *Confirm Options*.  To *Confirm Options*, review the information presented and ensure that it is all correct.  Click *Submit*.

Reviewer: ```suggestion also not certain if the last sentence is talking specifically about a UI element or screen name or the action of confirming options, if it is specifically the UI element then use bold, of talking about the action of confirm options then reword and not emphasis and use sentence capitalization i.e. "to confirm options, ..."
For the **Storage**, **Scaling/Upgrading Policy** and **Advanced DNS Settings** you can use the defaults for basic installation by clicking **Next** or bypassing and going directly to **Confirm Options**.  To *Confirm Options*, review the information presented and ensure that it is all correct.  Click *Submit*.
```

![SCALENextcloudConfirmOptions](/images/SCALE/SCALENextcloudConfirmOptions.png "SCALE Nextcloud Confirm Options")

> The Nextcloud application will now be present on the *Installed Applications* page.  Select *Portal* and you will be directed to the *Nextcloud* login page within your browser.

Reviewer: ```suggestion
The Nextcloud application is now present on the **Installed Applications** page.  Select **Portal** and you are directed to the **Nextcloud** login page within your browser.
```

![SCALENextcloudLogin](/images/SCALE/SCALENextcloudLogin.png "SCALE Nextcloud Login")

> Enter the credentials specified during installation and click *Log in*.  You will be directed to the *Nextcloud Hub*.

Reviewer: ```suggestion
Enter the credentials specified during installation and click *Log in**.  You are directed to the **Nextcloud Hub**.
```
